src/NzbDrone.Core/NetImport/TMDb/TMDbSettings.cs

```diff
@@ -69,6 +69,10 @@ public TMDbSettings()
         [FieldDefinition(0, Label = "TMDb API URL", HelpText = "Link to to TMDb API URL, do not change unless you know what you are doing.")]
         public string Link { get; set; }
 
+        [FieldDefinition(0, Label = "Account ID", HelpText = "Your account ID for Radarr API.")]
+//        I have built a simple tool at https://github.com/elChapoSing/tmdb_account_id to get it but not sure how to integrate it.
```

Comment:
You could include a link to something with more information (e.g. a wiki page).

Reply:
ideally Radarr should include a way to get the account_id.
Yes you need to have the user identify on tmdb.
My repo demonstrates the proper way to get it based on tmdb v4 documentation (https://developers.themoviedb.org/4/auth/user-authorization-1) but properly integrating it into Radarr is beyond my skill level I'm afraid.
If you think adding it as a parameter that the user has to fill is enough then just linking to my repo in the help tooltip should be enough ? I will make sure the readme there is well laid out.